pep-0001.txt

```diff
@@ -659,6 +660,9 @@ archives, rather than just a mailto: or hyperlink to the list itself.
 The Type header specifies the type of PEP: Standards Track,
 Informational, or Process.
 
+The optional Topic header specifies which topic, if any, the PEP belongs in.
```

Comment:
```suggestion
The optional Topic header lists which special topic(s),
if any, the PEP belongs under.
```

* PEPs can (and very occasionally do) list multiple topics
* Clarify these are _special_ topics, rather than something more general (at least, right now)
* "Belongs under" in this context, not "belongs in"

Reply:
Thanks, I used this but used "topics" instead of the slightly unsightly "topic(s)"

Reply:
Yeah, I figured you might not be so fond of that—in that event, given it's such a rare case, I actually think we're better off with the original then. I'll submit another suggestion.